resources/views/log-history.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layout')
+
+@section('title', 'Movies List')
+
+@section('content')
+    <div class="container">
+        <table class="table">
+            <thead>
+            <tr>
+                <th scope="col">Дата входа</th>
+                <th scope="col">Пользователь</th>
+                <th scope="col">IP</th>
+            </tr>
+            </thead>
+            @foreach($visits as $visit)
+                <tr>
+                    <td>{{$visit->enter_time?->format('G:i:s Y/m/d -D')}}</td>
```

Comment:
тут наверное не нужно делать null safe
у тебя же всегда есть время в базе
```suggestion
                    <td>{{$visit->enter_time->format('G:i:s Y/m/d -D')}}</td>
```